webapp/webpack.config.js

```diff
@@ -36,8 +66,12 @@ module.exports = {
         'react-router-dom': 'ReactRouterDom',
     },
     output: {
+        devtoolNamespace: PLUGIN_ID,
```

Comment:
I think we should add some other loaders such as the scss loader. Adding dev dependencies like that doesn't increase bundle size, so there's no bloat from adding it.

`npm install --save-dev style-loader css-loader sass-loader`

Then in the `webpack.config.js` [rules](https://github.com/mattermost/mattermost-plugin-jira/blob/dd988f94ebcf834806aeabefc470b410977dc187/webapp/webpack.config.js#L48-L64):

```js
{
    test: /\.scss$/,
    use: [
        'style-loader',
        {
            loader: 'css-loader',
        },
        {
            loader: 'sass-loader',
            options: {
                includePaths: ['node_modules/compass-mixins/lib', 'sass'],
            },
        },
    ],
}
```

Reply:
Great suggestions! Updated.